src/lib/mapping.js

```diff
@@ -70,7 +70,8 @@ function mapTeamList (row) {
     description: row.description,
     path: row.path,
     organizationId: row.org_id,
-    usersCount: parseInt(row.users_count)
+    usersCount: parseInt(row.users_count),
+    total: row.total
```

Comment:
I was also thinking that at the end we'll make everything consistent. But we should establish the exact structure in this discussion.
And another point: if we rely only on Joi for data validation.